### DISCUSSION TITLE
electron/

### DISCUSSION THREAD

**Discussion by giscus** [OweQian/oweqian.github.io#9] [General]
# electron/

项目地址： ElectronVue3 构建开发环境 按如下步骤搭建开发环境： 创建项目 首先通过命令行创建一个 Vue 项目： 1 npm create vite@latest electron -- --template vue-ts 安装 electron 依赖： 1 npm i electron -D 调整 package.json 文件： 1 2

https://oweqian.xyz/electron/

**Comment by EricZZZ** [comment]
大工程呦😝

**Reply by OweQian** [reply]
😝新的一年，努力搬砖
